extras/rya.benchmark/pom.xml

```diff
@@ -106,11 +106,29 @@
                 </executions>
                 <configuration>
                     <!-- Place the generated source within the 'src' directory so license-maven-plugin will find it. -->
-                    <outputDirectory>src/main/gen</outputDirectory>
+                    <outputDirectory>src/gen/java</outputDirectory>
                     <packageName>org.apache.rya.benchmark.query</packageName>
                 </configuration>
             </plugin>
             
+            <plugin>
```

Comment:
Plugin execution is unnecessary if code is stored in ${project.build.directory}/generated-sources.